packages/flutter_tools/lib/src/fuchsia/fuchsia_sdk.dart

```diff
@@ -0,0 +1,46 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import '../base/common.dart';
+import '../base/context.dart';
+import '../base/process.dart';
+
+/// The fuchsia SDK.
+FuchsiaSdk get fuchsiaSdk => context[FuchsiaSdk];
+
+/// The fuchsia SDK shell commands.
+///
+/// This workflow assumes development within the fuchsia source tree, providing
+/// the `fx` command-line tool.
+class FuchsiaSdk {
+  /// Invokes the `netaddr` command and return the output.
+  ///
+  /// This returns the network address of an attached fuchsia device.
```

Comment:
What if there are multiple attached devices?

Reply:
I don't really know - I could setup another device and see what the output is, but I was planning on punting on this for now

Reply:
That's fine - just figured the docs should explicitly state whether it's supported.